src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/data-broker-profiles/removal-under-maintenance/RemovalUnderMaintenanceView.tsx

```diff
@@ -77,35 +65,29 @@ export const RemovalUnderMaintenanceView = (props: Props) => {
         );
         return;
       }
+      void confetti();
 
       // Mark the current scan result as manually resolved
-      firstScanResultNotResolved.manually_resolved = true;
-
-      const nextUnresolved = props.data.results.find(
-        (scanResult) => !scanResult.manually_resolved,
+      const currentResultIndex = props.data.results.findIndex(
```

Comment:
Instead of marking `firstScanResultNotResolved.manually_resolved` as `true` we can assume that the previous result has been resolved and just get the next one.